pkg/reconciler/instances/istio/reset/pod/handler.go

```diff
@@ -45,8 +45,6 @@ type NoActionHandler struct {
 }
 
 func (i *NoActionHandler) Execute(object CustomObject, wg GetSyncWG) {
-	defer wg().Done()
```

Comment:
After removing this `defer` please also remove unused `wg GetSyncWG` function parameter, regenerate mocks for this type and update tests.